envsetup.sh

```diff
@@ -426,6 +406,16 @@ if [ -z "$DOCKER_REPO" ]; then
   export DOCKER_REPO=yoedistro/yoe-build:stretch
 fi
 
+# if we are building using docker, we don't really care what /bin/sh is since the Yoe docker images defaults
+# to using bash for /bin/sh. If user is building in their host system, then require that /bin/sh be bash
+if [ "$DOCKER_REPO" = "none" ]; then
+  if [ "$(readlink /bin/sh)" != "bash" ]; then
+    echo "ERROR: /bin/sh should point to bash, other shells (such as dash) have not been extensively tested"
+    echo "on Ubuntu/debian, try: sudo dpkg-reconfigure dash"
+    return 1
+  fi
+fi
+
```

Comment:
this looks ok. We dropped checking for 'pv' package. So I think that still is needed otherwise ```yoe_install_image``` function to write SD-Card will not work.

Reply:
we have another check for pv later in the file:

```
yoe_check_install_dependencies() {
  if ! pv -V 1>/dev/null; then
    echo "ERROR: Please install the pv utility (http://www.ivarch.com/programs/pv.shtml)"
    return 1
  fi
}

# write a WIC image to media (SD, USB, etc)
yoe_install_image() {
  DRIVE=$1
  IMAGE_NAME=$2

  yoe_check_install_dependencies || return 1

  if [ ! $DRIVE ] || [ ! $IMAGE_NAME ]; then
    echo "Usage: yoe_install_image /dev/sdX image_name"
    echo "WARNING!!!, make sure you specify your SD card and not a workstation disk"
    echo
    return 1
  fi
```

Does that cover all the cases?

Reply:
ah superb. I think that should do it.